python/dgl/heterograph.py

```diff
@@ -4701,6 +4701,7 @@ def int(self):
                               self._node_frames,
                               self._edge_frames)
 
+
```

Comment:
I would still prefer having a `def reverse(...)` here for backward compatibility since we have `DGLGraph.reverse`.